src/mtool/ui/src/components/VolumeManagement/CreateVolume/index.js

```diff
@@ -675,29 +652,12 @@ class CreateVolume extends Component {
                 </Tooltip>
               </FormControl>
             </Grid>
-            {/* <Grid item container xs={12} sm={6} justifyContent="flex-end" className={classes.formControl}>
-              <FormControl className={classes.volumeName}>
-                  <FormControlLabel
-                    control={
-                      <Checkbox name="mount_vol_checkbox" color="primary" id="mount-vol-checkbox"
-                        checked={this.state.mount_vol}
-                        value="Mount Volume"
-                        onChange={this.handleChange}
-                      />
-                    }
-
-                    label="Mount Volume"
-                    className={classes.labelCheckbox}
-                  />
-              </FormControl>
-            </Grid> */}
             <Grid
               item
               container
               xs={12}
               display="flex"
-              justifyContent="space-between"
-              className={`${classes.volBtnContainer} ${classes.formControl}`}
+              className={classes.volBtnContainer}
             >
               <Tooltip
                 title="Please wait... Volume creation is in progress. It may take anywhere between few seconds to few minutes"
```

Comment:
In my humble opinion, we can make the second sentence simpler?

How about "It may take from a few seconds to few minutes"?

Reply:
Okay thanks for the suggestion, will change the title.